Makefile

```diff
@@ -27,3 +28,8 @@ lint: docker-build
 
 docker-build:
 	docker build . --tag $(DOCKER_IMAGE)
+
+update-versions:
```

Comment:
Mhhh, I would not do it way. I would imitate https://github.com/akeneo/pim-helpcenter#dropdown-to-choose-the-documentation-version. I would generate if from Github branches.

Fetching data from the production server  to update the file is weird.

Anyway it was GTM in https://github.com/akeneo/pim-docs/pull/1540, so not blocking for this pull up.

Reply:
Im not sure there is an ideal way to handle this! 
I wrote this feature...  with automation in mind, and because 5.0 did not show.  The script can handle an emtpy file.
Using git was an option, but how could i was afraid I might bump into inconsistencies!
Another option is to list remote folders....

Reply:
What we chose on another project is to version the json on master and only publish it on master from master branch. 
The advantage of that is: 
- the state is defined by the repository. Not by repository + production.
- it's centralized (changing one things such as adding a label/traduction implies to modify every branches here).


The advantage of your solution is that you don't have to deactivate this workflow when creating a new branch such as 6.0.

:+1: let's try with it, it's merged anyway :)